modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -18,6 +18,8 @@ any questions.
 
 - [Codemod](#codemod)
   - [Instructions](#instructions)
+- [Component Updates](#component-updates)
+  - [Styling API and CSS Tokens](#styling-api-and-css-tokens)
```

Comment:
Do we want to structure this in the way we did before where we just add the PR links or the individual links within the Table of Contents to each component? Just curious.

Reply:
ah, is it not like the way it was before?